src/components/EditComponents.tsx

```diff
@@ -199,14 +210,28 @@ return(
             icon={<IconEdit/>}
           />
         </Box>
-        <div style={{display: 'flex', gap: '3rem', flexWrap: 'wrap'}}>
-          {newListInfo.slice(0,-1).map((info, index) => (
-           <Segment key={index} segmentData={info} onClick={() => onSegmentClick(info.id)} />))}
+        <br />
```

Comment:
`<br>` Elemente sollten nur in Fliesstext für Textumbrüche verwendet werden und nicht zur Positionierung von Elementen. Versuche mit `padding` und `margin` dieses Ziel zu erreichen.